app/build.gradle

```diff
@@ -24,9 +24,12 @@ dependencies {
     implementation 'com.android.support.constraint:constraint-layout:1.1.2'
     implementation 'com.android.support:design:27.1.1'
     implementation 'com.android.support:cardview-v7:27.1.1'
+    implementation 'com.google.firebase:firebase-auth:11.6.0'
```

Comment:
Just as question... is there a specific reason we are not using the last version (16.0.2) ??